stdlib/http/build.gradle

```diff
@@ -1,122 +0,0 @@
-    implementation project(':ballerina-crypto')
-    implementation project(':ballerina-logging')
-    implementation project(':ballerina-lang:annotations')
-    implementation project(':ballerina-mime')
-    implementation project(':ballerina-config')
-    implementation project(':ballerina-config-api')
-    implementation project(':ballerina-system')
-    implementation project(':ballerina-task')
-    implementation project(':ballerina-stringutils')
-    implementation project(':ballerina-observability')
-    testCompile project(':ballerina-cli')
-    implementation project(':ballerina-observability-internal')
-    testCompile 'org.slf4j:slf4j-simple'
-
-    balaImplementation project(path: ':ballerina-lang:annotations', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-auth', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-cache', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-config-api', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-crypto', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-file', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-io', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-log-api', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-mime', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-reflect', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-runtime-api', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-system', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-time', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-stringutils', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-observability', configuration: 'balaImplementation')
-    balaImplementation project(path: ':ballerina-observability-internal', configuration: 'balaImplementation')
-    interopImports project(':ballerina-config-api')
-    interopImports project(':ballerina-io')
-    interopImports project(':ballerina-log-api')
-    interopImports project(':ballerina-system')
-    interopImports project(':ballerina-task')
-    interopImports project(':ballerina-time')
-    interopImports project(':ballerina-reflect')
-    interopImports project(':ballerina-runtime-api')
-    interopImports project(':ballerina-mime')
-
-    // transitive
-    balaImplementation project(path: ':ballerina-task', configuration: 'balaImplementation')
-
-    interopImports project(':ballerina-io')
-
-    implementation 'io.netty:netty-codec-http'
-    implementation ('org.wso2.transport.http:org.wso2.transport.http.netty') {
```

Comment:
There are some netty dependencies in io.netty:netty-codec-http javaProject.gradle file. Shall we remove them too?

Reply:
Ack. Will remove and see.

Reply:
We can fix with a separate PR as well.

Reply:
I think we'll need to take a closer look at these. Seems to be in used in tests and test utils.